tests/manager/test_queryfilter.py

```diff
@@ -0,0 +1,135 @@
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import Sequence, Column, Integer, String
+import sqlalchemy as sa
+
+from ai.backend.manager.models.minilang.queryfilter import Query2sql
+
+test_json = r'''
+User(and:{
+        eq:{
+            name:"tester"
+        },
+        gt:{
+            age:20
+        }
+    }) {
+    name,
+    age
+}
+'''
+
+test_deep_json = r'''
+User(and:{
+        or:{
+            eq:{
+                name:"tester"
+            },
+            eq:{
+                name:"lablup"
+            }
+        },
+        gt:{
+            age:20
+        }
+    }) {
+    name,
+    age
+}
+'''
+
+escape_filter = "or:{eq:{name:\"test\'er ♪\"},gt:{age:20}}"
+underbar_filter = "eq:{full_name:\"tester1\"}"
+
+Base = declarative_base()
+Session = sessionmaker()
+engine = create_engine('sqlite:///:memory:', echo=True)
+Session.configure(bind=engine)
+conn = engine.connect()
+q2s = Query2sql()
+
+
+class User(Base):
+    __tablename__ = 'users'
+
+    id = Column(Integer, Sequence('user_id_seq'), primary_key=True)
+    name = Column(String(50))
+    full_name = Column(String(50))
+    age = Column(Integer)
+
+    def __init__(self, name, full_name, age):
+        self.name = name
+        self.full_name = full_name
+        self.age = age
+
+    def __repr__(self):
+        return "<User('%s', '%s', '%s')>" % (self.name, self.full_name, self.age)
+
+
+Base.metadata.create_all(engine)
+session = Session()
+
+session.add_all([
+    User('tester', 'tester1', 30),
+    User('test\'er', 'tester2', 30),
+    User('test\'er ♪', 'tester3', 20)])
+
+session.commit()
```

Comment:
Put these as a fixture, instead of module-level codes, so that it is executed only when tests that actually requires the dummy db, and not when the test framework imports this module.